public/package.json

```diff
@@ -5,8 +5,8 @@
   "homepage": "",
   "author": "Bitovi",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "install": "node build.js"
+    "install": "node build.js",
+    "test": "testee test.html --browsers firefox --reporter Spec"
```

Comment:
I would have assumed we'd use Phantom since we can add it as an NPM dependency. Firefox may not be installed by the developer.